### PR TITLE
Fix PerformInteraction with VR only

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Implements/Fixes https://adc.luxoft.com/jira/browse/AAW-862

### CODE REVIEW

File: ffw/UIRPC.js [hunk 1]
```diff
@@ -391,6 +391,12 @@ FFW.UI = FFW.RPCObserver.create(
                 SDL.ResetTimeoutPopUp.resetTimeoutRPCs.removeObject('VR.PerformInteraction');
               }
               array['UI.PerformInteraction'] =  request.params.timeout/1000 * increase_value;
+
+              if (request.params && request.params.choiceSet == null) {
```
Comment (AKalinich-Luxoft) [inline ffw/UIRPC.js:403]: @LitvinenkoIra are you sure it is a proper condition to remove the UI part?

File: ffw/UIRPC.js [hunk 2]
```diff
@@ -391,6 +391,12 @@ FFW.UI = FFW.RPCObserver.create(
...
+                console.log('remove UI');
```
Comment (AKalinich-Luxoft) [inline ffw/UIRPC.js]: @LitvinenkoIra looks like a debug message, please remove it

File: ffw/UIRPC.js (hunk 1 above)
Comment (LitvinenkoIra) [reply ffw/UIRPC.js:403]: @AKalinich-Luxoft Here we need to check that Performinteraction from mobile was sent with VR_ONLY mode. So according to this diagram https://luxproject.luxoft.com/confluence/display/APPLINK/PerformInteraction#PerformInteraction-1.15.2.RequestsUI/VR-portionsprocessing I made an assumption that UI should be sent without choiceSet param in this case

File: ffw/UIRPC.js (hunk 2 above)
Comment (LitvinenkoIra) [reply ffw/UIRPC.js]: Done in d70c4290f1d80552bd0f5febdf197873f436002c

**Pull request merged**